lego/apps/frontpage/views.py

```diff
@@ -52,12 +54,23 @@ def list(self, request):
                 request.user, queryset_events_base
             )
 
+        try:
```

Comment:
You can also do something like this, so you don't have to do `try-except` and the extra if.

```
poll_qs = Poll.objects.filter(pinned=True).order_by("created_at").last()
poll = DetailedPollSerializer(poll_qs, _context_stuff_).data if poll_qs else None
```

Reply:
Nice! Thats alot cleaner